csfieldguide/chapters/content/en/complexity-and-tractability/sections/tractability.md

```diff
@@ -16,7 +16,7 @@ If you are interested, you can [download the spreadsheet here](files/tractable-s
 There's a very simple rule that computer scientists use to decide if an algorithm is tractable or not, based on the complexity (estimated number of steps) of the algorithm.
 Essentially, if the algorithm takes an exponential amount of time or worse for an input of size *n*, it is labelled as intractable.
 This simple rule is a bit crude, but it's widely used and provides useful guidance.
-(Note that a factorial amount of time, *n!*, is intractable because it's bigger than an exponential function.)
+(Note that a factorial amount of time, *n!*, is intractable because it's bigger than an exponential function).
 
 To see what this means, let's consider how long various algorithms might take to run.
 The following interactive will do the calculations for you to estimate how long an algorithm might take to run.
```

Comment:
This, and sentences elsewhere including the above teacher note, appear to refer to the interactive that was deleted further down. If the interactive is no longer there the text should be updated to reflect that